server/src/main/java/org/eclipse/lsp/cobol/core/preprocessor/delegates/GrammarPreprocessorListenerImpl.java

```diff
@@ -65,11 +64,14 @@
 
   private static final String HYPHEN = "-";
   private static final String UNDERSCORE = "_";
-  private static final String SYNTAX_ERROR_CHECK_COPYBOOK_NAME = "Syntax error by GrammarPreprocessorListenerImpl#checkCopybookName: ";
+  private static final String SYNTAX_ERROR_CHECK_COPYBOOK_NAME =
+      "Syntax error by GrammarPreprocessorListenerImpl#checkCopybookName: ";
+  private static final String WS = " ";
```

Comment:
Aren't some of those constants defined in the `ProcessingConstants` class?

Reply:
Done. Got rid of WS